src/bootctl/bootctl-status.c

```diff
@@ -511,12 +513,17 @@ int verb_status(int argc, char *argv[], void *userdata) {
                         if (loader_url)
                                 printf(" Net Boot URL: %s\n", loader_url);
 
+                        if (sysfail_entry)
+                                printf("System Failure Reason: %s\n", sysfail_reason);
```

Comment:
the fields are aligned to the `:`. please stick to that. i.e. shorten System Failure to SysFail?